src/main/webapp/index.html

```diff
@@ -1,48 +1,37 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Air Quality Map</title>
-    <link rel="stylesheet" href="/style.css">
-    <script src="/script.js"></script>
-  </head>
-  <body>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Air Quality Map</title>
+  <link rel="stylesheet" href="/style.css">
+  <script src="/script.js"></script>
+  <script defer
+    src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBwXfHdwfNsdIYmwRfViMkPMPfxNUXOzCQ&libraries=visualization&callback=initMap">
+    </script>
+</head>
+
+<body>
+  <div id="map"></div>
+
+  <div class="main">
     <!-- Sidebar -->
     <div id="sidebar">
       <!-- The top "nav" in the sidebar -->
-      <div class="topnav"> 
+      <div class="topnav">
         <div class="search-container">
           <input type="text" placeholder="Search a location.." name="search" id="location-search-bar">
           <button class="button Search" id="location-search-button" onClick="searchLocation()">Search</button>
         </div>
-        <button class="button directions" id="directions" onClick="showRoutes()">Directions</button> 
+        <button class="button directions" id="directions" onClick="showRoutes()">Directions</button>
         <button class="button directiones" id="close-directions" onClick="hideRoutes()">Close directions</button>
-      </div> 
+      </div>
+      
       <div id="location-info"></div>
     </div>
     <button id="close-sidebar" onClick="closeSidebar()">close sidebar</button>
```

Comment:
These could be folded down into a single button with a "Toggle" behaviour.

Currently, the button floats off in the middle of things when the sidebar is closed; can it migrate to the edge when the sidebar isn't open?

Reply:
Done